src/model/article.py

```
@@ -99,3 +99,11 @@ class Article(Base):
         nullable=True,
         default=None,
     )
+
+    attachments: Mapped[list["Attachment"]] = relationship(
```

Ruff (0.15.9)

[error] Undefined name `Attachment`

(F821)